dockerfile

```diff
@@ -4,4 +4,6 @@ WORKDIR /app
 RUN dotnet restore
 WORKDIR /app/src/Web.Application
 RUN dotnet restore
-WORKDIR /app/src/Web.Application
\ No newline at end of file
+WORKDIR /app/tests/Tests
+RUN dotnet restore
+RUN dotnet test --logger "trx;LogFileName=unit_tests.xml"
```

Comment:
I think if we run xunit tests we will use dotnet xunit (https://xunit.github.io/docs/getting-started-dotnet-core), it has more options and can produce various types of reports